lib/css-url-fixer.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+module.exports = (idMap, value, name="") => {
```

Comment:
Can you rename variables for more clarity.

value -> attributeValueOrCSS
name -> attributeName.

Also, add a comment about what it does:

"Handle all SVG references correctly, which can be

a) via attributes: `url(#abc)`
b) via <use> tags: `<use href="#abc" />`
c) via css: `.class { fill: url(#abc) }`
"

Reply:
This isn't in lint rules, but I prefer if there's a space `name = ""` :)

Reply:
Sound good, I will do those changes.